devel/Dockerfile

```diff
@@ -57,6 +58,23 @@ RUN if [ "$TARGETPLATFORM" != "linux/arm/v7" ]; then \
         cd ../.. && rm -rf qemu; \
     fi
 
-ENV PATH="/opt/phoenix/utils:${PATH}"
+# install Jlink tools for 32-bit armv7 architecture
```

Comment:
nice, probably we would like to add jlink for host (x86_64) builds in the future also) - no need to change anything right now, but we might want to use generic `/opt/SEGGER/JLink/` for binaries regardless of the host architecture (that way most of the code below would be shared)